corehq/apps/reports/commtrack/standard.py

```diff
@@ -90,6 +91,12 @@ class CurrentStockStatusReport(GenericTabularReport, CommtrackReportMixin):
     ]
     exportable = True
     emailable = True
+    ajax_pagination = True
+    asynchronous = True
+
+    @property
+    def total_records(self):
+        return LedgerES().domain(self.domain).count()
```

Comment:
This should have the filters added. `LedgerES().domain(self.domain).section(section_id).case(case_ids).count()` (also add entry_id if necessary.

Also this could be in the same esaccessor file as the other queries are

Reply:
Done here:
https://github.com/dimagi/commcare-hq/pull/20876/commits/61135e2efe73f4e32cd2f529f0c4157d68f8564b